src/MovieCard.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react'
 import './MovieCard.css'
+// import { getMovieDetails } from './MoviePage'
+
+const handleClick = (event) => {
+    console.log(event.target.id)
+}
 
 const MovieCard = ( {id, title, poster_path, backdrop_path, average_rating, release_date} ) => {
+    
     return (
-        <div className="Movie-card" id={id}>
+        <div className="Movie-card" id={id} onClick={handleClick}>
```

Comment:
I was thinking more about this after we logged off last night... I wonder if the move is to put our `onClick={}` on the parent container (`MovieContainer`). I could be way off base here- but am thinking back to Vanilla JS stuff where we would put listeners on the parent container to make each element clickable...